lib/Sodium/FFI.pm

```diff
@@ -2,7 +2,7 @@ package Sodium::FFI;
 use strict;
 use warnings;
 
-our $VERSION = '0.009';
+our $VERSION = '0.010';
```

Comment:
Can you put this one back, please? The tooling takes care of version bumps and the current Git version (0.009) is the next to be released.

Reply:
I should really add a CONTRIBUTING.md doc here so this is all makes sense as the dzil tooling can be easily overlooked. I'll try to get that done as soon as I can.

Reply:
Fixed